### ISSUE TITLE
Dropdown menu cropped in table layout (version 14.30.0)

### ISSUE THREAD

**Issue by taiwanleaftea** [orchidsoftware/platform#2881]
**Describe the bug**

I have dropdown action menu built with this code

<img width="797" alt="Снимок экрана 2024-08-21 в 15 49 08" src="https://github.com/user-attachments/assets/15903e2e-05a4-4ff5-8ca1-9afd8887888b">

Until version 14.27.2 everything worked fine. After update to the 14.30.0 the dropdown is cut by next table row.

<img width="367" alt="Снимок экрана 2024-08-21 в 15 44 46" src="https://github.com/user-attachments/assets/5d5fd6ec-457c-400e-8b21-146cd8f6fb66">

**Expected behavior (version 14.27.2)**

<img width="350" alt="Снимок экрана 2024-08-21 в 15 54 48" src="https://github.com/user-attachments/assets/107013c2-24bc-4514-91a5-03a3f9f7de69">

**Desktop (please complete the following information):**

 - OS: MacOS 13.6.9
 - Browser Firefox 129.0.2, Safari 17.6

**Comment by tabuna** [comment]
I added the ability to make the entire table row clickable. To achieve this, I wrapped the `<tr>` element with position: relative and added an `<a>` element that fills the entire row.

Unfortunately, this solution doesn't work in Safari, both on desktop and mobile. Safari does not support positioning for table rows, as noted in this bug report https://bugs.webkit.org/show_bug.cgi?id=240961.

Neither the Bootstrap framework nor various blog tricks were able to resolve the issue, and several attempts at finding a working solution were unsuccessful. However, everything should now be working correctly in modern browsers.
